src/upload/UploadPage.jsx

```diff
@@ -35,9 +35,9 @@ export class UploadPage extends React.Component {
       defaultOpen
     >
       <div className="container">
-        <div className="row pb-2">
+        <div className="row mb-3 mb-md-2 ">
           <div className="col-md">
-            <div className="btn btn-primary w-100">
+            <div className="btn btn-primary w-100 mb-1">
```

Comment:
This adds a little space between the buttons when they are stacked. Before, they would touch.

Old:
![image](https://user-images.githubusercontent.com/11871801/72929945-d59c5500-3d28-11ea-9535-c2c0ceeabc26.png)


New:
![image](https://user-images.githubusercontent.com/11871801/72929909-c5847580-3d28-11ea-9b8a-1269fe624365.png)